manifests/fips.pp

```diff
@@ -0,0 +1,58 @@
+# Manages FIPS
+#
+# @params use_fips Boolean
+#   If enabled, the system will be FIPS 140-2 enabled.
+#
+# @params use_fips_aesni Boolean
+#   If enabled and $use_fips is true, then install dracut-fips-aesni
+#
+class simp::fips (
```

Comment:
I think this might need to be its own module.

I know that it sounds silly for this much code but, as things expand into new OSs, this is going to get quite expansive and we will want to use it independently.